ibis/expr/types/relations.py

```diff
@@ -666,6 +666,68 @@ def select(
 
     projection = select
 
+    def local_write(table: ir.TableExpr, name: str | None = None) -> ir.TableExpr:
+        """Create a local-write of a table.
+
+        Parameters
+        ----------
+        table
+            Table expression for the local-write
+        name
+            Name of local file or files for the write
+
+        Returns
+        -------
+        TableExpr
+            A table expression with the same schema
+        """
+        node = ops.LocalWrite(table, name=name)
+        return node.to_expr()
+
+    def as_of_merge(
```

Comment:
Is there a reason to have `as_of_merge` and `asof_join`? Why doesn't the existing one work for you?

Reply:
Not related to UDT; please skip.